CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## Unreleased
+
+### Fixes
+
+- Fix Coroutine Context Propagation using CopyableThreadContextElement, requires `kotlinx-coroutines-core` version `1.6.1` or higher ([#2838](https://github.com/getsentry/sentry-java/pull/2838))
```

Comment:
```suggestion
- Fix Coroutine Context Propagation using CopyableThreadContextElement ([#2838](https://github.com/getsentry/sentry-java/pull/2838))
  - `sentry-kotlin-extensions` now requires `kotlinx-coroutines-core` version `1.6.1` or higher
```

^ I presume it won't even compile with older versions, right? This should make it more clear.

What options do we have to keep compatibility? Shipping two separate modules / JARs?

Reply:
Kotlin version is at 1.8 for a while already, this is more about the kotlinx-coroutines-core dependency version rather

Reply:
Yeah we could make it part of v7, sounds like a great idea 👍 @lbloder mind re-targeting this to `feat/7.0.0` branch?

Reply:
Absolutely! Makes sense to have this in the next major. PR is re-targeted. Thanks for the input @adinauer , @marandaneto and @romtsn

Reply:
I guess we have to merge feat/7.0.0 against main to avoid that many changes in this PR. I'll do that 👍